workspaces/code-coverage/plugins/code-coverage/src/api.ts

```diff
@@ -59,7 +59,7 @@ export class CodeCoverageRestApi implements CodeCoverageApi {
   private async fetch<T = unknown | string | JsonCoverageHistory>(
     path: string,
   ): Promise<T | string> {
-    const url = await this.discoveryApi.getBaseUrl('code-coverage');
+    const url = await this.discoveryApi.getBaseUrl('codeCoverage');
```

Comment:
I'm assuming this still works with the old backend?

Reply:
I'm not sure.  I took the suggestion from the last comment in the linked issue.  I'm new to Backstage, have only used the new backend.

Reply:
Should this be a minor version upgrade with an update to the front end plugin as well? That way no existing integrations get effected?

Reply:
Isn't that exactly what this is?

Reply:
we need to make sure both old and new backend and frontends work - if they do I'm good with it as is

Reply:
DO you have any recommendations for how I test that?